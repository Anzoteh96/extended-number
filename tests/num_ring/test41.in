-933651652514315544417582829162973050672950900045666450104043656592868847883472659967760436737860471011990284434907207264194
519721765664872678745029891938668600394143394072912965702616729909208290178480695116373162110341471806788636580081189232410
-53661245214073531821694235004284455391616309627513470745670599166287961163061867674861037230416863792489794272271234626240
