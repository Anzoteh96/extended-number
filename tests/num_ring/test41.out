Enter the number base b:
Enter two numbers n and m:
b is 933651652514315544417582829162973050672950900045666450104043656592868847883472659967760436737860471011990284434907207264194
n mod b is 519721765664872678745029891938668600394143394072912965702616729909208290178480695116373162110341471806788636580081189232410
m mod b is 879990407300242012595888594158688595281334590418152979358373057426580886720410792292899399507443607219500490162635972637954
n + m is 466060520450799146923335656934384145002527084445399494956946130742920329015418827441512124879924608014298842307809954606170
n - m is 573383010878946210566724126942953055785759703700426436448287329075496251341542562791234199340758335599278430852352423858650
n * m is 481564753132502689822697416632298395053565194042478699368350768856337191115987790019573610079438540789813801553993619843256
n ** m is 801277053213211772395968942646961359144125085875813983670634368949584073427274624862606007663012191561374771545353007916564
